_local/scripts/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Any reason why the shebang for `build-git.sh` and `entrypoint.sh` are different than the ones for `test-*`?